### ISSUE TITLE
WebTorrent can't open from icon

### ISSUE THREAD

**Issue by tycho01** [tycho01/nix-config#10]
```
$ ls /nix/store | grep torrent-desktop
$ cat /nix/store/yv8dpjqy6vy3qypnyw88140zbbppi8pf-webtorrent-desktop-0.20.0/share/applications/webtorrent-desktop.desktop
[...]
Path=/opt/webtorrent-desktop
Exec=/nix/store/yv8dpjqy6vy3qypnyw88140zbbppi8pf-webtorrent-desktop-0.20.0/bin/WebTorrent %U
```

These could come from a [dynamic](https://github.com/webtorrent/webtorrent-desktop/blob/41511c56153b878b0dd39b2adac9525fb66fb35c/static/linux/webtorrent-desktop.desktop#L11-L13
) or a [static](https://github.com/webtorrent/webtorrent-desktop/blob/41511c56153b878b0dd39b2adac9525fb66fb35c/static/linux/share/applications/webtorrent-desktop.desktop#L11-L13) file.

This `Exec` seems fine, but the `Path=/opt/webtorrent-desktop` doesn't exist, and appears [used](https://specifications.freedesktop.org/desktop-entry-spec/latest/ar01s06.html) "to run the program in", whatever that means.

The nix seems to [replace](https://github.com/NixOS/nixpkgs/blob/77645d6a018a627f9380bbd689431664638b43d3/pkgs/applications/video/webtorrent_desktop/default.nix#L74-L76) the desktop binary path:
```
      # Fix the desktop link
      substituteInPlace $out/share/applications/webtorrent-desktop.desktop \
        --replace /opt/webtorrent-desktop/WebTorrent $out/bin/WebTorrent
```
That seems to have gone right.

**Comment by tycho01** [comment]
Let's reopen if #14 won't solve.

**Comment by tycho01** [comment]
upgraded. rechecking.

**Comment by tycho01** [comment]
wanna try older version, but blocked by #34.

**Comment by tycho01** [comment]
now, the [only nix change](https://github.com/NixOS/nixpkgs/blame/77645d6a018a627f9380bbd689431664638b43d3/pkgs/applications/video/webtorrent_desktop/default.nix#L74-L76) just upgraded 0.19.0 -> 0.20.0.

**Comment by tycho01** [comment]
copying the `.desktop` elsewhere then launching it yields:

> There was an error launching the application.
> Details: Failed to change to directory "/opt/webtorrent-desktop" (No such file or directory)

... which appears correct.
Commenting the offending `Path` line(s) does in fact fix the issue.

**Comment by tycho01** [comment]
PR at https://github.com/NixOS/nixpkgs/pull/45477

**Comment by tycho01** [comment]
PR merged! Now I'll just need to upgrade, see #34 or #37.

**Comment by tycho01** [comment]
Impatient for upgrades to fix this, I manually added a desktop shortcut in https://github.com/tycho01/nix-config/commit/3fecb5712a319fab68ef8de53ff7219cb14fbf34.
